spec/index.bs

```diff
@@ -2242,8 +2238,8 @@ typedef [EnforceRange] unsigned long GPUTextureUsageFlags;
 interface GPUTextureUsage {
     const GPUFlagsConstant COPY_SRC          = 0x01;
     const GPUFlagsConstant COPY_DST          = 0x02;
-    const GPUFlagsConstant SAMPLED           = 0x04;
-    const GPUFlagsConstant STORAGE           = 0x08;
+    const GPUFlagsConstant SHADER_READ       = 0x04;
+    const GPUFlagsConstant SHADER_WRITE      = 0x08;
```

Comment:
This means I can have a texture with only `SHADER_WRITE` that's bound to a `var t : [[access(read)]] texture_storage_2d<r32float>`?

Reply:
RO storage textures are removed from WGSL as well, so you can't have that

Reply:
Weird, so we'll have `write`, `read_write` in the future (with potentially coercions between them) but not `read`?

Reply:
Would you need `SHADER_READ | SHADER_WRITE` for read_write, or just `SHADER_WRITE`? If the latter, maybe `SHADER_READ`/`SHADER_READWRITE`?

Reply:
Never mind, I see you addressed this while I was writing the comment :)

Reply:
@Kangz correct, yes. Slightly weird for people coming from GLSL/Vulkan (not for anybody else), but I think it's ok.

@kainino0x we talked about this on office hours earlier today, and concluded that `STORAGE` needs to be the name for the mutable usage, still, instead of `SHADER_WRITE`. So once read-write is added, it's still in `STORAGE` usage, and it's less confusing.